hermes-schema/src/main/java/pl/allegro/tech/hermes/schema/SchemaVersionsResponse.java

```diff
@@ -0,0 +1,44 @@
+package pl.allegro.tech.hermes.schema;
+
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+
+public class SchemaVersionsResponse {
```

Comment:
Not sure if `Response` is the best suited name here. Perhaps `Result` would be better?

Reply:
Done